pkg/network/setup/network.go

```diff
@@ -84,8 +84,8 @@ func (v VMNetworkConfigurator) getPhase1NICs(launcherPID *int, networks []v1.Net
 			return nil, fmt.Errorf("no iface matching with network %s", networks[i].Name)
 		}
 
-		// Binding plugin and SR-IOV devices are not part of the phases
-		if iface.Binding != nil || iface.SRIOV != nil {
+		// Binding plugin, SR-IOV and Slirp devices are not part of the phases
+		if iface.Binding != nil || iface.SRIOV != nil || iface.Slirp != nil {
```

Comment:
This is confusing.
We could have filtered these out even without this PR change. Did something stopped working when keeping this here? If not, I would do it independently of this change, through a separate commit or even a PR.

Reply:
Slirp interface has to be filtered at this point otherwise the network setup code is invoked where it shouldn't.